Library/OcDebugLogLib/OcLog.c

```diff
@@ -522,6 +685,19 @@ OcConfigureLogProtocol (
       Private->OcLog.FileSystem   = LogRoot;
       Private->OcLog.FilePath     = LogPath;
 
+      //
+      // Write filters into Private.
+      //
+      FlexFilters = NULL;
+      if (AsciiStrCmp (LogModules, "*") != 0 && AsciiStrCmp (LogModules, "") != 0) {
+        LogModulesCopy = AllocateCopyPool (AsciiStrSize (LogModules), LogModules);
+        if (LogModulesCopy != NULL) {
+          FlexFilters = OcStringSplit (LogModulesCopy, L',', FALSE);
+          FreePool (LogModulesCopy);
```

Comment:
Noooo. Not with OcStringSplit based on what I wrote, as you have it.

Either change OcStringSplit to make a copy of each token it finds, or else you have to keep LogModulesCopy around and free it when you free FlexFilters.

Reply:
https://github.com/acidanthera/OpenCorePkg/pull/322/commits/7d9c977f4793ad0385db91b1369252bb5fd9b8f4